src/items.rs

```diff
@@ -25,6 +27,54 @@ use syntax::print::pprust;
 use syntax::parse::token;
 
 impl<'a> FmtVisitor<'a> {
+    pub fn visit_let(&mut self, local: &ast::Local, span: Span) {
+        self.format_missing_with_indent(span.lo);
+
+        // New scope so we drop the borrow of self (context) in time to mutably
+        // borrow self to mutate its buffer.
+        let result = {
+            let context = self.get_context();
+            let mut result = "let ".to_owned();
+
+            // 1 = ;
+            // FIXME(#82): reserving space for the semicolon overly conservative
+            // for multi-line expressions.
```

Comment:
It was about the comment, I certainly misunderstood it. Could you explain a little bit ?
Also, the following lines (42 & 43) seems thought only for cases without initialization nor type specified (only `- 1`, seems to be `;`, but we could nedd `- 2` for ` =`).

For integer sizes, while I'm not an expert of that king of things, I think the integer sizes doesn't matter a lot: we don't have large arrays of them, so memory space isn't an issue. Computation speed won't probably change a lot, as we use native machine integer size.
Indeed `u16` should be large enough, but it would force an explicit cast at each use of a string method.
If you think it might be a significant gain of performance, I would be happy to see a benchmark.